README.md

```diff
@@ -1,268 +1,78 @@
-![](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
+![freeCodeCamp Guide](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
 
 # freeCodeCamp Guide
-The freeCodeCamp community is building a massive, searchable "Guide." This reference tool will eventually include thousands of articles that cover all areas of development, design, and data science - all written to be easily understood by people new to coding.
 
-This repo is where we plan and maintain these Guide articles, which we then host on our community's wiki-like [Guide website](https://guide.freecodecamp.org).
+[![Build Status](https://img.shields.io/travis/freeCodeCamp/guides/master.svg?style=flat-square)](https://travis-ci.org/freeCodeCamp/guides) [![license](https://img.shields.io/github/license/freeCodeCamp/guides.svg?style=flat-square)](https://opensource.org/licenses/BSD-3-Clause)  [![Gitter](https://img.shields.io/gitter/room/freeCodeCamp/freeCodeCamp.svg?style=flat-square)](https://gitter.im/freeCodeCamp/freeCodeCamp)
```

Comment:
We might want to change the Gitter link to the [Contributors](https://gitter.im/FreeCodeCamp/Contributors) room rather than the general chat room.

Reply:
Done!